lua/nvim-treesitter/install.lua

```diff
@@ -361,10 +367,22 @@ local function install_lang(lang, ask_reinstall, cache_folder, install_folder, w
       return
     end
 
-    local yesno = fn.input(lang .. " parser already available: would you like to reinstall ? y/n: ")
-    print "\n "
-    if not string.match(yesno, "^y.*") then
-      return
+    -- Now check if the ABI is not correct
+    local parser = inspect_lang(lang)
```

Comment:
there shouldn't be a inspect_lang before install.

Reply:
Yup, that's why we need the lockfile thing now, I am going to change that

Reply:
Rewording: at this point, we know that the parser is installed, because of the `is_installed` call before hand.